packages/vaadin-icons/gulpfile.js

```diff
@@ -6,33 +6,41 @@ const modify = require('gulp-modify');
 const cheerio = require('cheerio');
 const concat = require('gulp-concat');
 
-gulp.task('icons', function () {
+function createCopyright() {
+  return `/**
+ * @license
+ * Copyright (c) 2015 - 2021 Vaadin Ltd.
```

Comment:
nit: why not generate the year dymanically?
```suggestion
 * Copyright (c) 2015 - ${new Date().getFullYear()} Vaadin Ltd.
```